ndk/src/native_activity.rs

```diff
@@ -120,6 +122,9 @@ impl NativeActivity {
     /// Path to the directory with the application's OBB files.
     ///
     /// Only available as of Honeycomb (Android 3.0+, API level 11+)
+    ///
+    /// # Safety
+    /// `obbPath` must be a pointer to a valid C-string.
```

Comment:
I'm wondering it it would make sense to drop the `unsafe` decoration on this one, similar to `external_data_path`.
`obbPath` is an implementation details so not that helpful for the user imo.

Reply:
Yeah I agree, it's a bit weird to demand an application to guarantee safety of internal pointers. Perhaps it was done this way because of `Only available as of Honeycomb (Android 3.0+, API level 11+)`? If so, `# Safety` should simply be added above this line, relying on the user to guarantee API 11+ through other means instead.